concent_api/core/message_handlers.py

```diff
@@ -954,6 +958,58 @@ def store_message(
     return stored_message
 
 
+def handle_send_subtask_results_verify(
+    request,
+    subtask_results_verify: message.concents.SubtaskResultsVerify
+):
+    report_computed_task = subtask_results_verify.subtask_results_rejected.report_computed_task
+    compute_task_def = report_computed_task.task_to_compute.compute_task_def
+    validate_id_value(
+        compute_task_def['subtask_id'],
+        'subtask_id'
+    )
+    client_public_key = decode_client_public_key(request)
+    other_party_public_key = decode_other_party_public_key(request)
+
+    if Subtask.objects.filter(
+        Q(subtask_id=compute_task_def['subtask_id']),
+        Q(state=Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name) |  # pylint: disable=no-member
+        Q(state=Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name),  # pylint: disable=no-member
+    ).exists():
+        return message.concents.ServiceRefused(
+            reason=message.concents.ServiceRefused.REASON.DuplicateRequest,
+        )
+    if not base.is_requestor_account_status_positive(request):  # pylint: disable=no-value-for-parameter
+        return message.concents.ServiceRefused(
+            reason=message.concents.ServiceRefused.REASON.TooSmallRequestorDeposit,
+        )
+    if subtask_results_verify.subtask_results_rejected.reason != SubtaskResultsRejected.REASON.VerificationNegative:
+        return message.concents.ServiceRefused(
+            reason=message.concents.ServiceRefused.REASON.InvalidRequest,
+        )
+
+    store_or_update_subtask(
+        task_id=compute_task_def['task_id'],
+        subtask_id=compute_task_def['subtask_id'],
+        provider_public_key=other_party_public_key,
+        requestor_public_key=client_public_key,
+        state=Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER,
+        next_deadline=subtask_results_verify.subtask_results_rejected.timestamp + settings.ADDITIONAL_VERIFICATION_CALL_TIME,
+        set_next_deadline=True,
+        task_to_compute=report_computed_task.task_to_compute,
+        subtask_results_rejected=subtask_results_verify.subtask_results_rejected
+    )
+
+    send_verification_request()
+    
+    encoded_client_public_key = b64encode(decode_client_public_key(request))
```

Comment:
it will `request['HTTP_CONCENT_CLIENT_KEY']` as `bytes`, ie:
`'ifLken5xe8tIgHerZqk0q8pzHYFsFRIwLaX+xSQeIeeYk5Bf9vOKBvJoNeeaiyTfaVso5ToBPPzzegFycrFY1A=='` -> 
`b'ifLken5xe8tIgHerZqk0q8pzHYFsFRIwLaX+xSQeIeeYk5Bf9vOKBvJoNeeaiyTfaVso5ToBPPzzegFycrFY1A=='`